src/main/kotlin/net/dungeonhub/application/commands/CreatePrivateVCCommand.kt

```
@@ -0,0 +1,122 @@
+                            allowed += Permission.MoveMembers
+                        }
+
+                        parentId = categoryId
+
+                    }
+
+                    respond {
+                        content = "${voiceChannel.mention} has been created successfully!"
+                    }
+
+                } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)